client/ionic/src/pages/About.module.css

```diff
@@ -0,0 +1,4 @@
+.page ion-header ion-toolbar {
+  --background: translucent;
+  --background: blue;
```

Comment:
not sure what the effect of double specifying this property is?  does it mean anything?

Reply:
removed in https://github.com/WorldHealthOrganization/app/pull/74